empress/topoSort/ReconBuilder.py

```diff
@@ -0,0 +1,277 @@
+# ReconBuilder.py
+# This code builds a representation of a host tree, parasite tree, and a reconfiguration
+# as specified in Recon.py using the host tree, parasite tree, and reconfiguration 
+# representation in the DTL.
+
+from empress.topoSort.Tree import NodeLayout
+from empress.topoSort.Tree import TreeType
+from empress.topoSort.TreeFormatConverter import dict_to_tree
+
+__all__ = ["build_trees_with_temporal_order"]
+
+def build_trees_with_temporal_order(host_tree, parasite_tree, reconciliation):
+    """
+    This is the main function for visualization purposes.
```

Comment:
I think you might want to change this comment to "Use topological sort to order the vertices inside host and parasite tree. The output trees can be used for visualization."